src/callbacks.jl

```diff
@@ -45,3 +32,202 @@ function reeval_internals_due_to_modification!(integrator::DDEIntegrator)
 
     integrator.u_modified = false
 end
+
+# track propagated discontinuities for dependent delays
+
+struct DiscontinuityCallback{F,D<:Discontinuity,A,R} <: AbstractContinuousCallback
+    lags::F
+    discontinuities::Vector{D}
+    interp_points::Int
+    atol::A
```

Comment:
should stick to always using abstol and reltol in the context of diffeq.

Reply:
I'll change it 👍